src/plotman/analyzer.py

```diff
@@ -17,21 +17,30 @@ def analyze(logfilenames, bytmp, bybitfield):
             phase_time = {}  # Map from phase index to time
             n_sorts = 0
             n_uniform = 0
+            is_first_last = False
 
             # Read the logfile, triggering various behaviors on various
             # regex matches.
             for line in f:
                 # Beginning of plot job.  We may encounter this multiple
                 # times, if a job was run with -n > 1.  Sample log line:
-                # Starting plotting progress into temporary dirs: /mnt/tmp/01 and /mnt/tmp/a
-                m = re.search(r'^Starting plotting.*dirs: (.*) and (.*)', line)
+                # 2021-04-08T13:33:43.542  chia.plotting.create_plots       : INFO     Starting plot 1/5
+                m = re.search(r'Starting plot (\d*)/(\d*)', line)
                 if m:
                     # (re)-initialize data structures
                     sl = 'x'         # Slice key
                     phase_time = {}  # Map from phase index to time
                     n_sorts = 0
                     n_uniform = 0
 
+                    seq_num = int(m.group(1))
+                    seq_total = int(m.group(2))
+                    is_first_last = seq_num == 1 or seq_num == seq_total
+
+                # Temp dirs.  Sample log line:
+                # Starting plotting progress into temporary dirs: /mnt/tmp/01 and /mnt/tmp/a
+                m = re.search(r'^Starting plotting.*dirs: (.*) and (.*)', line)
```

Comment:
Can't wait for this to get caught up on a path like `C:\Users\Jane and John\Desktop\Chia`.  `:]`  Not that I have a proposed change...  so don't worry about it.

Reply:
Yeah.........